.github/workflows/label-notify.yml

```
@@ -14,4 +14,5 @@ jobs:
                   team/web=@joelkw @felixfbecker @jeanduplessis
                   team/cloud=@tsenart
                   team/search=@lguychard
+                  team/search=@pooja
```

These can be on the same line, and it should be your github username for notifications :) 

```suggestion
                  team/search=@lguychard @poojaj-tech
```

---

Thanks for the review! I was living in an alternate reality where I had @pooja as my github username 😄